server/routers/usersRouter.js

```diff
@@ -33,40 +33,18 @@ router.get("/", (req, res) => {
       res.status(500).json({ error: "The users could not be retrieved." });
     });
 });
+/****** Add a User ******/
+router.post('/', async (req, res) => {
+    const { user }= req.body
 
-signin = (req, res) => {
-  //implementing user signin
-  const creds = req.body;
-  users
-    .get(creds.username)
+    await users.insert(req.body)
     .then(user => {
-      //Check to see is username is valid
-      if (user && bcrypt.compareSync(creds.password, user.password)) {
-        const token = generateToken(user);
-        res.json({ id: user.id, token });
-      } else {
-        res.status(404).json({ err: "Invalid username or password" });
-      }
+    res.status(200).json(user)
```

Comment:
I think this should be status code 201? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

Reply:
I can change this. I just wanted to make sure I was getting it returned at the time. Wasn't thinking about the status code meaning. Easily changeable thanks.

Reply:
Fixed this it is outdated please re-review thank @bcabanayan